config/dev.yaml

```diff
@@ -11,189 +11,246 @@ common:
-    how_to_get_help: "if this is prod, ask someone to fix the how_to_get_help setting"
+    sentry_dsn:
 
-    # Help text given when a dropdown permission argument is available in
-    # permission request. This is probably help around how this dropdown was
-    # populated and how to get what the user wants if they don't see it as an
-    # option.
+background:
+    # Sentry DSN for logging exceptions.
+    #
     # Type: str
-    permission_request_dropdown_help:
+    sentry_dsn:
 
-    # Site-specific docs.
-    # Type: list of maps giving url, name, and description strings.
-    site_docs:
+    # How long to wait between iterations.
+    #
+    # Type: int
+    sleep_interval: 60
 
-    # Sentry DSN for logging exceptions
-    # Type: str
-    sentry_dsn:
+ctl:
+    # Directories for one-offs. If set, load oneoffs from these directories
+    # which are run via grouper-ctl.
+    #
+    # Type: List[str]
+    oneoff_dirs: []
 
-    # A list of lists of the shells that users are allowed to select. The first argument is
-    # the shell location (i.e. /bin/bash), and the second is a user readable format for the
-    # shell
-    shell:
-        - [/bin/false, Your administrator has not configured Grouper for shell management]
+    # Module paths for one-offs to load from.
+    #
+    # Type: List[str]
+    oneoff_module_paths:
+      - oneoffs.refresh_public_keys
+      - oneoffs.check_public_keys
 
-    # The email domain that all service accounts must have
+fe:
+    # The IP address to listen to requests on, or leave empty to listen to
+    # all addresses.
+    #
     # Type: str
-    service_account_email_domain: "svc.localhost"
-
-api:
-    # Number of worker processes to fork for receving requests. This option
-    # is mutually exclusive with debug.
-    # Type: int
-    num_processes: 1
+    address: "127.0.0.1"
 
     # The port to listen to requests on.
+    #
     # Type: int
-    port: 8990
+    port: 8989
 
-    # The IP address to listen to requests on, or leave empty to listen to
-    # all addresses.
+    # This is the prefix URL used to access external resources hosted by a
+    # mirror of CDNJS. This allows you to point at an internal mirror if you
+    # prefer.
+    #
     # Type: str
-    address: "127.0.0.1"
+    cdnjs_prefix: "//cdnjs.cloudflare.com"
 
-    # Passing debug option down tornado. Useful for development to
-    # automatically reload code.
+    # Passing debug option down Tornado. Useful for development to automatically
+    # reload code.
+    #
     # Type: bool
     debug: true
 
-    # How often to pull cache data from database in seconds.
+    # How to get help from the people who run this Grouper deployment. Should be
+    # in the form of an imperative sentence.
+    # https://en.wikipedia.org/wiki/Sentence_function#Imperative
+    # For example:"email grouper-admin@example.com"
+    #
+    # Type: str
+    how_to_get_help: "if this is prod, ask someone to fix the how_to_get_help setting"
```

Comment:
heh

Reply:
For the record, this was pre-existing -- I just moved it around. :)